### PR TITLE
Enhancment of scsi_add_remove test script

### PR DESCRIPTION
**Pull request by Naresh-ibm**
existing test script cannot be run on vfc disk whic does not has pci_address.
and also, script doesn't has verification part after remove or added the scsi
disk. So modified the script to run using wwid os disk and also added the
verification part to it.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[scsi_add_remove_job_log_v3.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/3369614/scsi_add_remove_job_log_v3.txt)

**Comment by narasimhan-v** [comment]
@Naresh-ibm When you open a new PR for changes, please close the older ones.
And take care of assignments and review request as per the old PR

File: io/disk/scsi_add_remove.py [hunk 1]
```diff
@@ -33,44 +36,62 @@ def setUp(self):
...
             self.cancel("lsscsi is not installed")
-        self.pci_device = self.params.get("pci_device", default=None)
-        if not self.pci_device:
-            self.cancel("Please provide PCI address for which you \
-                        want to run the test")
+        self.wwids = self.params.get('wwids', default='').split(',')
+        system_wwids = multipath.get_multipath_wwids()
+        for wwid in self.wwids:
+            if wwid not in system_wwids:
+                self.cancel("%s not present in the system" % wwid)
```
Comment (narasimhan-v) [inline io/disk/scsi_add_remove.py]: I am still not fine with the assumption that it is only multipath devices that the test considers.

```
# multipath -ll
Jul 10 12:13:59 | DM multipath kernel driver not loaded

# lsscsi -u
[0:0:0:0]    disk    5000c500a8f849f9                  /dev/sda 
[1:0:0:0]    disk    5000c500a8f61901                  /dev/sdb 
[N:0:33:1]   disk    335256304a6002060025384100000001           /dev/nvme0n1
[N:0:33:26]  disk    335256304a600206002538410000001a           /dev/nvme0n2
```

**Comment by Naresh-ibm** [comment]
closing this in favour of #1220

**Pull request closed without merging**